etl/steps/data/garden/wb/2023-12-29/gender_statistics.meta.yml

```diff
@@ -13,30 +13,68 @@ tables:
+        description_processing: This indicator was created by OWID by combining paternity leave and parental leave father quota indicators. For more details on each indicator see **How producer describes this data**."
         description_key:
           - This indicator is created by combining two specific types of parental leave. These components, together, provide a detailed understanding of the total leave available to fathers in relation to childbirth and early childcare.
           - The first component, the length of paid paternity leave, focuses on the leave specifically designated for fathers at the time of childbirth. This includes the days available exclusively to fathers to support them in bonding with the newborn and assisting in early childcare. This leave plays a crucial role in supporting the father's engagement during the early stages of a child's life.
           - The second component, the length of paid parental leave for father, includes days of parental leave that are specifically reserved for fathers. These are often defined by "use-it-or-lose-it" policies or fathers’ quotas. This part of the indicator represents additional support provided to fathers, extending beyond the traditional paternity leave period. It ensures that fathers have dedicated time for child-rearing, promoting their active involvement in early childcare and supporting gender equality in parental responsibilities.
-          - By integrating these two elements, the indicator provides a total count of the calendar days of leave available to fathers. It reflects a country's legislative approach to supporting fathers in the workforce and their role in early childcare. This integration highlights the legal and policy frameworks in place to ensure paternal involvement and support in early child-rearing. This measure is crucial for assessing how different countries prioritize paternal involvement, postnatal care, and gender equality in parental leave policies.
+          - The sum of these components reflects the total days of leave available specifically for the father, highlighting the support for paternal involvement and gender equality in parenting.
+          - This indicator uses standardized assumptions, like the woman having one child and residing in the largest business city, to ensure comparability, though this approach may not capture variations in laws affecting women in different states, rural areas, or minority groups
+
         processing_level: major
         display:
           numDecimalPlaces: 0
+      total_maternity_leave_for:
+        title: Total days of leave available for the mother
+        unit: days
+        short_unit: ''
+        description_processing: This indicator was created by OWID by combining maternity leave and parental leave mother quota indicators. For more details on each indicator see **How producer describes this data**.
```

Comment:
... maternity leave, parental leave mother quota and shared parental leave indicators...